libraries/joomla/database/driver/postgresql.php

```diff
@@ -1624,4 +1624,21 @@ public function quoteBinary($data)
 	{
 		return "decode('" . bin2hex($data) . "', 'hex')";
 	}
+
+	/**
+	 * Internal function to get the name of the default schema for the current PostgreSQL connexion.
```

Comment:
```suggestion
	 * Internal function to get the name of the default schema for the current PostgreSQL connection.
```
We use English, not French in comments ;-)